config/routes.rb

```diff
@@ -86,6 +89,7 @@
     end
   end
 
+  mount Sidekiq::Web => "/sidekiq"
```

Comment:
@knutsenm @andrewroth I'm assuming we don't expose a sidekiq route like this?

Reply:
Usually there's an admin check around it..

Reply:
Yes; Global Registry, for example, uses Basic Auth:
https://github.com/CruGlobal/global_registry/blob/master/config/initializers/sidekiq.rb#L44

Reply:
@knutsenm I do not have access to the line of code you mention there... how do you say to implement it? is the next the idea you expressed plus some basic auth somewhere in controllers?

```ruby
  namespace :admin do
    mount Sidekiq::Web => "/sidekiq"
  end
```

Reply:
No; by "admin check" he meant authorization against some secret, like an API key for instance.

https://github.com/sidekiq/sidekiq/wiki/Monitoring#authentication has some examples.

Reply:
done